README.md

```diff
@@ -256,6 +256,28 @@ const client = new Client(authClient);
 const result = await client.createInvoice({amount: 1000});
 ```
 
+#### Handling refresh token 
+Access tokens do expire. If an access token is about to expire, this library will automatically use a refresh token to retrieve a fresh one. Utilising the tokens event is a simple approach to guarantee that you always save the most recent tokens:
```

Comment:
Could we add a note here: make sure to only initialize one instance of the client per token pair at a time, otherwise you could lose access to the user's token.

@bumi for backends that are scaled to more than one instance, this could still be an issue, right?

Reply:
resolved